docs/plugin_fundamentals/import_automation.md

```diff
@@ -160,7 +149,7 @@ The input fields are defined via properties with the annotation `[ConfigurationI
 It is also possible to create your own configuration elements if the predefined elements are not sufficient. See [Custom UI]({% link docs/advanced_topics/ui.md %}).
 
 ## IImportRunner
-IImportRunner is responsible for processing the cyclical import and reacting to problems and errors accordingly. In our example, the plug-in connects to our PiWeb Cloud instance and checks for the presence of the defined import target part name. In addition, the part is created if it is not found. The import loop also checks the import source and creates new measured values under the defined part.
+Is responsible for processing the cyclical import and reacting to problems and errors accordingly. In our example, the Auto Importer connects to our PiWeb Cloud instance and checks for the presence of the defined import target part name. In addition, the part is created if it is not found. The import loop also checks the import source and creates new measured values under the defined part.
```

Comment:
In the end it is the Auto Importer that connects, but the function to connect has to be explicitly programmed in the plug-in, there is no build-in call for it, so I had originally changed from “Auto Importer connects” to “plug-in connects”, but I am fine with both versions.

Reply:
Fixed.